sopel/cli/config.py

```diff
@@ -0,0 +1,161 @@
+# coding=utf-8
+"""Sopel Config Command Line Interfaces (CLI): ``sopel-config``"""
+from __future__ import unicode_literals, absolute_import, print_function, division
+
+import argparse
+import os
+
+from sopel import tools, config
+from . import utils
+
+
+def build_parser():
+    """Configure an argument parser for ``sopel-config``"""
+    parser = argparse.ArgumentParser(
+        description='Sopel configuration tool')
+
+    # Subparser: sopel-config <sub-parser> <sub-options>
+    subparsers = parser.add_subparsers(
+        help='Actions to perform',
+        dest='action')
+
+    # sopel-config list
+    list_parser = subparsers.add_parser(
+        'list',
+        help='List available configurations from Sopel\'s homedir',
```

Comment:
Not just here, but the only place I'll leave a note:

Are we so opposed to using double-quotes for strings that we won't even do so to avoid escaping `'`?

Reply:
Ah ah, no, we can totally do double quotes here.